src/Util/Logfile/AbstractReader.php

```diff
@@ -34,11 +36,34 @@ private static function getReaders()
             return static::$readers;
         }
 
+        static::$readers = static::getCustomReaders();
         static::$readers[] = new ApacheCommonLogFormatReader();
 
         return static::$readers;
     }
 
+    public static function autoloadCustomReaders($clazz)
+    {
+        $parts = explode('\\', $clazz);
+        $path = __DIR__.DIRECTORY_SEPARATOR.'Custom'.DIRECTORY_SEPARATOR.end($parts).'.php';
+        if (is_file($path)) {
+            require $path;
+        }
+    }
+
+    private static function getCustomReaders()
+    {
+        $finder = Finder::create()->in(__DIR__.DIRECTORY_SEPARATOR.'Custom');
+        array_map(array($finder, 'name'), array('*.php'));
```

Comment:
Could you rewrite this line to `$finder->name('*.php');` ?

Reply:
fixed, thanks for the review